datasplits/datasplit_dcm-brno_seed50.yaml

```diff
@@ -0,0 +1,156 @@
+test:
+- sub-1860B6472B
+- sub-2295B4676B
+- sub-2336B
+- sub-2347B6454B
```

Comment:
Just to confirm, @naga-karthik. `dcm-brno` contains both HC and MildCompreession/DCM. 
Subjects with a single session (i.e., short participant_id), e.g. `sub-2336B` are HC. While subjects with two sessions (i.e., long participant_id), e.g., `sub-2347B6454B` are MildCompreession/DCM.
Are you taking this into account when creating your "(mega) dataset stats" in https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/pull/125#issue-2655187987, aren't you?

Reply:
hey @valosekj, if the corresponding `participants.tsv` has `HC` for `sub-236B` (and similarly for other short participant ids), then, yes, they are automatically taken care of!

Reply:
Nice, thanks for the confirmation!